selfdrive/test/test_onroad.py

```diff
@@ -26,7 +26,7 @@
   "./loggerd": 45.0,
   "./locationd": 9.1,
   "selfdrive.controls.plannerd": 22.6,
-  "./_ui": 15.0,
```

Comment:
@deanlee any idea why this went up after the recent refactors?

Reply:
imo, the new OnroadHud window Increased the workload of window compositing. 
we may need to consider using plugin(https://doc.qt.io/qt-5/embedded-linux.html) or QGraphicsScene(with opengl viewport) to reduce the cpu usage if necessary.